index.html

```diff
@@ -22,68 +22,68 @@
             <img
-              src="./src/assets/plus-minus.png"
+              src="./src/assets/plus-minus.svg"
               alt="plus-minus"
               class="gray-color__button-text"
             />
           </button>
-          <button class="buttons__button-numbers gray-color">
+          <button class="buttons__buttonNumbers gray-color">
             <img
-              src="./src/assets/percent.png"
+              src="./src/assets/percent.svg"
               alt="percent"
               class="gray-color__button-text"
             />
           </button>
-          <button class="buttons__button-numbers purple-color">
+          <button class="buttons__buttonNumbers purple-color">
             <img
               src="./src/assets/divide.png"
               alt="divide"
               class="purple-color__button-text"
             />
           </button>
-          <button class="buttons__button-numbers dark-gray-color">7</button>
-          <button class="buttons__button-numbers dark-gray-color">8</button>
-          <button class="buttons__button-numbers dark-gray-color">9</button>
-          <button class="buttons__button-numbers purple-color">
+          <button class="buttons__buttonNumbers dark-gray-color">7</button>
+          <button class="buttons__buttonNumbers dark-gray-color">8</button>
+          <button class="buttons__buttonNumbers dark-gray-color">9</button>
+          <button class="buttons__buttonNumbers purple-color">
             <img
               src="./src/assets/multiplication.png"
               alt="multiplication"
               class="purple-color__button-text"
             />
           </button>
-          <button class="buttons__button-numbers dark-gray-color">4</button>
-          <button class="buttons__button-numbers dark-gray-color">5</button>
-          <button class="buttons__button-numbers dark-gray-color">6</button>
-          <button class="buttons__button-numbers purple-color">
+          <button class="buttons__buttonNumbers dark-gray-color">4</button>
+          <button class="buttons__buttonNumbers dark-gray-color">5</button>
+          <button class="buttons__buttonNumbers dark-gray-color">6</button>
+          <button class="buttons__buttonNumbers purple-color">
             <img
               src="./src/assets/minus.png"
               alt="minus"
               class="purple-color__button-text"
             />
           </button>
-          <button class="buttons__button-numbers dark-gray-color">1</button>
-          <button class="buttons__button-numbers dark-gray-color">2</button>
-          <button class="buttons__button-numbers dark-gray-color">3</button>
-          <button class="buttons__button-numbers purple-color">
+          <button class="buttons__buttonNumbers dark-gray-color">1</button>
+          <button class="buttons__buttonNumbers dark-gray-color">2</button>
+          <button class="buttons__buttonNumbers dark-gray-color">3</button>
```

Comment:
maybe will be better to rename these class names to buttons__numbers? why do we need the "button" word twice?

Reply:
ps. you can do it in the next PR